client/src/style/home/home.pcss

```diff
@@ -0,0 +1,22 @@
+.home {
+  @apply grid place-content-center text-center;
+}
+
+.home__box {
+  @apply bg-amber-100 p-10 items-center content-center;
+}
+
+.home__header {
+  @apply p-3;
+}
+
+.home__sub-header {
+  @apply p-3
+}
+
+/* unsure of where to put this. Had issues with it loading from main.pcss, but I may be using it in other places in my app */
```

Comment:
oh interesting... main.pcss would definitely be ideal. I can try to help you troubleshoot why it wasn't working in main.pcss, but we might need to ask Dan instead, since this isn't a strong point of mine

Reply:
looking at main.pcss, I wonder if it would work if it weren't nested in @layer base? But I don't know why that would make a difference or fully understand what @layer base is doing